simpleclient_servlet/src/main/java/io/prometheus/client/filter/MetricsFilter.java

```diff
@@ -149,6 +155,10 @@ public void init(FilterConfig filterConfig) throws ServletException {
                 .help(help)
                 .name(metricName)
                 .register();
+
+        statusCounter = Counter.build(metricName + "_status", "HTTP status codes of " + help)
```

Comment:
Counters should end in _total

Having a label name in a metric name is generally an anti-pattern, as it won't make sense if that label is aggregated away. It may be okay here though.

Reply:
I remove **status** from the metric name. It will now look like this
```
# HELP http_request_status HTTP status codes of The time taken fulfilling servlet requests
# TYPE http_request_status counter
http_request_total{path="/my/api/1",method="GET",status="200",} 4.0
http_request_total{path="/my/api/2",method="GET",status="500",} 2.0
```

Reply:
That'd end up kinda clashing with the histogram though, which isn't desirable.

Reply:
I added **status** back in. It will now look like this.
```
# HELP http_request_status HTTP status codes of The time taken fulfilling servlet requests
# TYPE http_request_status counter
http_request_status_total{path="/my/api/1",method="GET",status="200",} 4.0
http_request_status_total{path="/my/api/2",method="GET",status="500",} 2.0
```

Reply:
I do want it removed, but we can't for other reasons :)